scripts/check_e2e_tests_are_captured_in_ci.py

```diff
@@ -0,0 +1,158 @@
+                utils.snake_case_to_camel_case(match.group().lower()))
+
+    return sorted(suites_from_jobs)
+
+
+def get_e2e_suite_names_from_script_travis_yml_file():
+    """Extracts the script section from the .travis.yml file.
+
+    Returns:
+        list(str). A list of names of test suites in script section in travis.yml file.
+    """
+    travis_file = read_and_parse_travis_yml_file()
+    script_raw = travis_file['script']
+    # The following line extracts the test suites from patterns like
+    # bash scripts/run_e2e_tests.sh --suite="accessibility"
+    hyphen_between_regex = re.compile(
+        r'(?<= bash scripts/run_e2e_tests.sh --suite=")(.*)(?=")')
+    suites_from_script = []
+
+    for script in script_raw:
+        matches = hyphen_between_regex.finditer(script)
+        for match in matches:
+            suites_from_script.append(match.group())
+
+    return sorted(suites_from_script)
+
+
+def get_e2e_suite_names_from_protractor_file():
+    """Extracts the test suites section from the .travis.yml file.
+
+    Returns:
+        list(str). A list of all test suites in protractor.conf.js file.
+    """
+    protractor_config_file = read_protractor_conf_file()
+    # The following line extracts suite object from protractor.conf.js.
+    suite_object_string = re.compile(
+        r'suites = {([^}]+)}').findall(protractor_config_file)[0]
+
+    # The following line extracts the keys/test suites from the key: value pair
+    # from the suites object.
+    key_regex = re.compile(r'\b(.*?)(?=:)')
+    protractor_suites = []
+    for match in key_regex.finditer(suite_object_string):
+        if len(match.group()) != 0:
+            protractor_suites.append(match.group())
+
+    return sorted(protractor_suites)
+
+
+def read_protractor_conf_file():
+    """Returns the contents of core/tests/protractor.conf.js file.
+
+    Returns:
+        str. The contents of protractor.conf.js, as a string.
+    """
+    protractor_config_file = python_utils.open_file(
+        os.path.join(
+            os.getcwd(), 'core', 'tests', 'protractor.conf.js'), 'r').read()
+    return protractor_config_file
+
+
+def read_and_parse_travis_yml_file():
+    """Returns the contents of .travis.yml, as a dict.
+
+    Returns:
+        dict. Contents of the travis.yml file parsed as a dict.
+    """
+    travis_ci_file = python_utils.open_file(
+        os.path.join(os.getcwd(), '.travis.yml'), 'r').read()
+    travis_ci_dict = utils.dict_from_yaml(travis_ci_file)
+    return travis_ci_dict
+
+
+def get_e2e_test_suites_to_exclude_from_travis_ci():
+    """Returns the names of test suites to exclude from Travis CI.
+
+    Returns:
+        list(str). A list of test suites that are excluded from Travis CI.
+    """
+    # These 4 test suites are not present in travis ci.
+    # One is extra ie. (full: [*.js]), and three other tests that
+    # are being run by circleCi.
+    return ['full', 'classroomPage', 'fileUploadFeatures', 'topicAndStoryEditor']
```

Comment:
Make this a module-level constant, and remove this function. Call it TEST_SUITES_NOT_RUN_ON_TRAVIS

Reply:
Done